modules/nf-core/picard/collecthsmetrics/main.nf

```diff
@@ -9,9 +9,10 @@ process PICARD_COLLECTHSMETRICS {
 
     input:
     tuple val(meta), path(bam), path(bai), path(bait_intervals, stageAs: "baits/*"), path(target_intervals, stageAs: 'targets/*')
-    tuple val(meta2), path(fasta)
-    tuple val(meta3), path(fai)
-    tuple val(meta4), path(dict)
+    tuple val(meta2), path(ref)
+    tuple val(meta3), path(ref_fai)
+    tuple val(meta4), path(ref_dict)
+    tuple val(meta5), path(ref_gzi) // ref_gzi only required if reference is gzipped
```

Comment:
This might be a silly question, but don't you need to set up some code if ref_gzi is not provided? Right now, in your tests you're providing it as empty, but it is there. If there are pipelines/workflows that use collecthsmetrics and don't provide even an empty ref_gzi, wouldn't they fail after your changes?

Or am I missing something?